tools/chromiumos/gen_autosuspend_rules.py

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env python3
-# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
please do not commit these changes (check docs/RELEASE)

Reply:
dropped.